lang/en/logstore_xapi.php

```diff
@@ -89,6 +89,7 @@
 $string['info'] = 'Info';
 $string['datetimegmt'] = 'Date/Time (GMT)';
 $string['logstorexapierrorlog'] = 'Logstore xAPI Error Log';
+$string['logstorexapihistoriclog'] = 'Logstore xAPI Historic Log';
```

Comment:
I'm not sure this is the correct name for it, but I'm not sure what it should be.
Does anyone else have any other ideas?

Reply:
Logstore xAPI - Error log's history

Reply:
This one is not for errors. It is for Historical logstore events that the user wants to search for to send.

Reply:
I see, then:
Logstore xAPI log's history

Reply:
Could it be "Logstore xAPI Historical Events Log" perhaps?